bqplot/marks.py

```diff
@@ -975,8 +975,9 @@ class Map(Mark):
 
     Attributes
     ----------
-    default_color: Color or None (default: None)
-        default color for items of the map when no color data is passed
+    colors: Dict (default: {'default_color': 'MediumSeaGreen'})
+        default colors for items of the map when no color data is passed. The dictionary should be indexed by the
```

Comment:
add a line specifying that the key `default_color` controls the color of the items for which no color was specified.